### PR TITLE
Release #1 3/3/2017

### PR DESCRIPTION
**Pull request by adelevie**

### CODE REVIEW

File: src/components/Form/DateControl/DateControl.jsx [hunk 1]
```diff
@@ -112,12 +113,12 @@ export default class DateControl extends ValidationElement {
         event.target.date = d
         super.handleChange(event)
 
-        // Always make sure the day is re-validated
-        if (['month', 'year', 'estimated'].includes(event.target.name)) {
-          this.refs.day.refs.input.focus()
-          this.refs.day.refs.input.blur()
-          event.target.focus()
-        }
+        // // Always make sure the day is re-validated
```
Comment (adelevie) [inline src/components/Form/DateControl/DateControl.jsx]: might want to delete these if they're commented out.

File: src/components/Form/DateControl/DateControl.jsx [hunk 2]
```diff
@@ -260,32 +261,39 @@ export default class DateControl extends ValidationElement {
   }
 
   render () {
+    console.log('month:', this.state.month)
```
Comment (adelevie) [inline src/components/Form/DateControl/DateControl.jsx]: does this need to be logged in production, or is this just for debugging?

File: src/components/Form/Dropdown/Dropdown.jsx [hunk 3]
```diff
@@ -104,15 +93,18 @@ export default class Dropdown extends ValidationElement {
...
-        valid = true
-        value = x.name
-      }
-    })
 
-    if (valid === false) {
-      errorCodes = 'notfound'
+    if (valid !== null) {
+      this.state.options.forEach(x => {
+        if (x.name.toLowerCase() === value.toLowerCase() || x.value.toLowerCase() === value.toLowerCase()) {
```
Comment (adelevie) [inline src/components/Form/Dropdown/Dropdown.jsx]: might want to name the variables and/or sweep into a named function

**Comment by adelevie** [comment]
LGTM

File: src/components/Form/DateControl/DateControl.jsx (hunk 2 above)
Comment (bmallred) [inline src/components/Form/DateControl/DateControl.jsx]: Yeah removed this in a recent commit on the `military-history` branch. Definitely was not meant to stick around.

File: src/components/Form/Dropdown/Dropdown.jsx (hunk 3 above)
Comment (bmallred) [inline src/components/Form/Dropdown/Dropdown.jsx]: Which variable do you mean? The `x` and/or `value`?

Comment (adelevie) [inline src/components/Form/Dropdown/Dropdown.jsx]: Yeah, I can't really figure out the meaning behind this check. What's it really checking for?

Comment (bmallred) [inline src/components/Form/Dropdown/Dropdown.jsx]: Cool... so like each `<option />` element has a `name` or `value` and it is going through comparing against the current `value` supplied by the input. If there is a match it places the value with the appropriate casing and such.

```
enter `az` get `Arizona`
enter `ariZoNa` get `Arizona`
```

**Pull request merged**